### PR TITLE
Fix #6552: Remove gravatar tooltip and update the Me screen design

### LINKED ISSUE #6552: Downgrade the Gravatar change popup tooltip
Back when the ability to change your Gravatar directly from the "Me" screen was introduced, we added a "New! Tap your Gravatar to change it" popup tooltip to help bring the feature into user's attention.

The feature is no longer new and we should probably remove the popup and replace it with a more subtle way to denote support for Gravatar changing.

### PR DESCRIPTION
**Pull request by maxme**
Fix #6552: Remove gravatar tooltip and update the Me screen design. Note: there is a bit of discussion on the issue itself about the design.

Before:

screenshot-2017-12-19_10 31 52 435

After:

screenshot-2017-12-19_10 25 12 167

### CODE REVIEW

**Comment by maxme** [comment]
cc @iamthomasbishop

File: WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java [hunk 1]
```diff
@@ -118,50 +110,6 @@ public void setUserVisibleHint(boolean isVisibleToUser) {
...
-                (getActivity(), R.color.color_primary));
-        mGravatarToolTipView = new ToolTipView.Builder(getActivity())
-                .withAnchor(mAvatarTooltipAnchor)
-                .withToolTip(toolTip)
-                .withGravity(Gravity.END)
-                .build();
-        mGravatarToolTipView.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                AnalyticsTracker.track(AnalyticsTracker.Stat.ME_GRAVATAR_TOOLTIP_TAPPED);
```
[the remark this answers is not in the document]
Comment (maxme) [reply WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java]: Removed in 3a45374b5b

File: WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java [hunk 2]
```diff
@@ -307,11 +245,10 @@ private void addDropShadowToAvatar() {
             mAvatarContainer.setOutlineProvider(new ViewOutlineProvider() {
                 @Override
                 public void getOutline(View view, Outline outline) {
-                    int padding = (mAvatarContainer.getWidth() - mAvatarImageView.getWidth()) / 2;
-                    outline.setOval(padding, padding, view.getWidth() - padding, view.getHeight() - padding);
+                    int padding = mAvatarContainer.getWidth() + 10;
+                    outline.setOval(0, 0, padding, padding);
```
[the remark this answers is not in the document]
Comment (maxme) [reply WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java]: There is a very narrow shadow, but I agree we should drop it. removed in 092a517115

File: WordPress/src/main/res/layout/me_fragment.xml [hunk 3]
```diff
@@ -13,67 +13,84 @@
...
-                    android:padding="@dimen/margin_large"
-                    android:indeterminate="true"
-                    android:clickable="true"
-                    android:visibility="gone"
-                    tools:visibility="visible"/>
-            </FrameLayout>
-
-            <org.wordpress.android.widgets.WPTextView
+                <org.wordpress.android.widgets.WPTextView
+                    android:id="@+id/change_photo"
```
[the remark this answers is not in the document]
Comment (maxme) [reply WordPress/src/main/res/layout/me_fragment.xml]: Yeah I forgot to wire that one 🤦‍♂️  - fixed in 092a517115

**Comment by maxme** [comment]
Comments addressed

> No progress bar nor change in the avatar. Not sure if this got broken in the PR or before it tbh.

I tried on a device and and emulator, it works well with both a local and remote image. Can you try to scope down the problem on your side?

**Comment by hypest** [comment]
OK, it seems that having the "Don't Keep Activities" debug setting ON breaks the normal operation of the feature and that's happening on `develop` as well so, not an issue of this PR.

So, LGTM!

**Pull request merged**